tools/json2bin/consts/pokemon/evo_methods.py

```diff
@@ -0,0 +1,42 @@
+from enum import Enum, Flag, auto
+
+
+class EvoMethod(Flag):
+    NONE                = 0
```

Comment:
I would prefix these with `EVO_` to explicitly define their purpose in the JSON files. We probably need to update the files to account for that, but that should be a simple script.